src/backend/joanie/tests/core/api/order/test_submit_installment_payment.py

```diff
@@ -371,7 +345,7 @@ def test_api_order_submit_installment_payment_with_not_matching_credit_card_id_i
         side_effect=DummyPaymentBackend().create_payment,
     )
     def test_api_order_submit_installment_payment_without_passing_credit_credit_card_id_in_payload(
-        self, _mock_create_payment, _mock_create_one_click_payment
+        self, mock_create_payment, _mock_create_one_click_payment
```

Comment:
the `_` prefix is a convention saying that this variable is not used and we know it. If the variable is used. do not use the `_` prefix. Check your test to fix this please.